### PR TITLE
perf: add Super 3.5 VL GB200 benchmark config

### PR DESCRIPTION
**Pull request by HuiyingLi**
# What does this PR do?

Add the measured Super 3.5 VL text-only benchmark configuration for four 4-GPU GB200 nodes in one NVLink domain.

# Changelog

- Configure EP16, 8K sequence length, local batch size 8, global batch size 128, and activation checkpointing.
- Use BF16 PyTorch fused Adam with separate TE expert parameter groups.
- Explicitly select text-backbone FLOPs with benchmark.flops_scope: text.
- Run 15 steps with 5 warmup steps and manual GC every 10 steps.
- Declare CI ownership, a 15-minute walltime, four nodes with four GPUs each, and the required allocator environment.
- Write results to `training_logs/super35_vl_text_8k_ep16_fused_adam.json`; W&B is disabled by default.

# Before your PR is "Ready for review"

- [x] Read and followed the contributor guidelines; commit includes DCO sign-off.
- [x] Validated the training settings with a four-node benchmark; YAML, new-recipe metadata, nightly references, and all 19 CI pipeline generation checks pass.
- [x] Documented the required allocator environment in the YAML.

# Additional Information

**Dependency:** #3901 adds the explicit text FLOPs scope used by this configuration.

The training settings match the configuration validated at `ebc691e9e`: **31.6911% average MFU** and **6.2763 seconds per step** over the final 10 steps. All four nodes exited successfully and the benchmark wrote its native JSON summary. The peak used for MFU is 2500 TFLOPs/s per GPU.

Launch with `PYTORCH_ALLOC_CONF=expandable_segments:True`. The model path is the prepared HSG configuration used for this measurement (88 text layers, MTP disabled); reproductions elsewhere must provide the equivalent model configuration.

### CODE REVIEW

**Review by akoumpa — APPROVED** [review]

File: examples/llm_benchmark/nemotron/super35_vl_text_8k_ep16_fused_adam.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  timeout_minutes: 10
+rng:
+  _target_: nemo_automodel.components.training.rng.StatefulRNG
+  seed: 42
+  ranked: true
+model:
+  _target_: nemo_automodel.NeMoAutoModelForCausalLM.from_config
+  config:
+    _target_: transformers.AutoConfig.from_pretrained
+    pretrained_model_name_or_path: /lustre/fsw/portfolios/coreai/users/huiyingl/super35_perf_hsg_20260915/model_full
```
Comment (akoumpa) [inline examples/llm_benchmark/nemotron/super35_vl_text_8k_ep16_fused_adam.yaml:40]: @HuiyingLi
